azure-pipelines.yml

```diff
@@ -136,36 +170,38 @@ jobs:
              "ToolVersion": "1.0"
            }
        ]
+    condition: and(succeeded(), startsWith(variables['SignArtifacts'], 'true'))
   - task: DeleteFiles@1
     displayName: 'Delete CodeSignSummary files'
     inputs:
       contents: '**\CodeSignSummary-*.md'
-  - task: CopyFiles@2
-    inputs:
-      SourceFolder: '..\..\..\buildoutput'
-      Contents: '*.nupkg'
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
-  - task: CopyFiles@2
-    inputs:
-      SourceFolder: '..\..\buildoutput'
-      Contents: '*.nupkg'
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
-  - task: CopyFiles@2
-    inputs:
-      SourceFolder: '$(Build.Repository.LocalPath)\buildoutput'
-      Contents: '*.zip'
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
-  - publish: $(Build.ArtifactStagingDirectory)
-    artifact: artifacts
+  - publish: $(Build.ArtifactStagingDirectory)\Functions.3.0.$(buildNumber)${{ variables.artifactSuffix }}.zip
+    artifact: Functions.3.0.$(buildNumber)${{ variables.artifactSuffix }}
+  - publish: $(Build.ArtifactStagingDirectory)\Functions.Private.3.0.$(buildNumber)${{ variables.artifactSuffix }}.win-x32.inproc.zip
+    artifact: Functions.Private.3.0.$(buildNumber)${{ variables.artifactSuffix }}.win-x32.inproc
+  - publish: $(Build.ArtifactStagingDirectory)\Functions.Symbols.3.0.$(buildNumber)${{ variables.artifactSuffix }}.win-x64.zip
+    artifact: Functions.Symbols.3.0.$(buildNumber)${{ variables.artifactSuffix }}.win-x64
+  - publish: $(Build.ArtifactStagingDirectory)\Functions.Symbols.3.0.$(buildNumber)${{ variables.artifactSuffix }}.win-x86.zip
+    artifact: Functions.Symbols.3.0.$(buildNumber)${{ variables.artifactSuffix }}.win-x86
+  - publish: $(Build.Repository.LocalPath)\packages\Microsoft.Azure.WebJobs.Script.3.0.$(buildNumber)${{ variables.artifactSuffix }}.nupkg
+    artifact: Microsoft.Azure.WebJobs.Script.3.0.$(buildNumber)${{ variables.artifactSuffix }}
+  - publish: $(Build.Repository.LocalPath)\packages\ExtensionsMetadataGenerator
+    artifact: ExtensionsMetadataGenerator${{ variables.artifactSuffix }}
+  - publish: $(Build.Repository.LocalPath)\packages\WebJobs.Script.Performance.App
```

Comment:
@brettsam @alrod  I made these updates so that there is one artifact per folder. 

Example artifacts: https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=4901&view=results (without suffix)
https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=4900&view=artifacts&type=publishedArtifacts (with suffix)